src/App.jsx

```diff
@@ -1,13 +1,20 @@
 import React from 'react';
+import { Route, Routes, Navigate } from 'react-router-dom';
 import { MovieProvider } from './context/MovieContext';
 
-import { Home } from './pages/Home/Home';
+import { Home, ErrorPage } from './pages';
 
 import './assets/styles/main.scss';
 
 const App = () => (
   <MovieProvider>
-    <Home />
+    <Routes>
+      <Route path='*' element={<ErrorPage />} />
+      <Route path='/search' element={<Home />} />
+      <Route path='/' element={<Navigate to='/search' replace />} />
+      <Route path='/search' element={<Home />} />
```

Comment:
I suppose it's a redudndant string because you already have the same on the 13 string, right?